utils/constants.py

```diff
@@ -7,3 +7,17 @@
 
 # maximum number of players that can be added
 MAX_NUM_PLAYERS_TEAM = 10
+
+TIER_VALUE = {
+    "IRON": 1,
+    "BRONZE": 2,
+    "SILVER": 3,
+    "GOLD": 4,
+    "PLATINUM": 5,
+    "DIAMOND": 6,
+    "MASTER": 7,
+    "GRANDMASTER": 8,
+    "CHALLENGER": 9,
+}
+
+RANK_VALUE = {"IV": 0, "III": 0.25, "II": 0.5, "I": 0.75}
```

Comment:
Just like how you did for `TIER_VALUE`, can we put `RANK_VALUE` in each line as well?

Reply:
The pylint auto formats it since the line is not long enough. I think it is easier to see if they are separate lines between keys and values but it keeps formatting it to one  line.

Reply:
This is fine, but let's keep it simple;
`I= 4`, `II=3`, `III=2`, `IV=1`.
And in rank_value calculations, just times it by 10th digit based on importance it would be easier to extend later on if we add different criteria (eg; lp point).
`rankvalue = 1000*TIER_VALUE + 100*RANK_VALUE`

Reply:
@dhmoon91 Got it, I'll change it. Another thing I hadn't consider for this as well as add summoner was not realizing that Master, GrandMaster and Challenger tiers don't have I,II,III,IV. I have created an issue to fix that as well.